src/main/java/org/folio/services/financedata/FinanceDataService.java

```diff
@@ -28,12 +55,133 @@ public Future<FyFinanceDataCollection> getFinanceDataWithAcqUnitsRestriction(Str
       .compose(effectiveQuery -> getFinanceData(effectiveQuery, offset, limit, requestContext));
   }
 
-  private Future<FyFinanceDataCollection> getFinanceData(String query, int offset, int limit, RequestContext requestContext) {
+  private Future<FyFinanceDataCollection> getFinanceData(String query, int offset, int limit,
+                                                         RequestContext requestContext) {
     var requestEntry = new RequestEntry(resourcesPath(FINANCE_DATA_STORAGE))
       .withOffset(offset)
       .withLimit(limit)
       .withQuery(query);
     return restClient.get(requestEntry.buildEndpoint(), FyFinanceDataCollection.class, requestContext);
   }
+
+  public Future<Void> putFinanceData(FyFinanceDataCollection financeData, RequestContext requestContext) {
```

Comment:
We can simplify implementation and accept only FyFinanceData instead of FyFinanceDataCollection
UI also will pass always only data for single particular FY
![image](https://github.com/user-attachments/assets/2fe29c63-dcb9-4500-bf8c-440a9d413795)

Reply:
and so we can simplify mod-finance-storage implementation as well